client/components/mma/cancel/cancellationReason.ts

```diff
@@ -37,6 +37,7 @@ export type CancellationReasonId =
 	| 'mma_cost_of_living'
 	| 'mma_cutting_subscriptions'
 	| 'mma_payment_issue'
+	| 'mma_price_increase'
 	| 'mma_article'
```

Comment:
we can remove the unused ones now? e.g. I don't think `mma_article` exists in this repo after this change

Reply:
yep good shout, I'll remove all the unused reasons 👍

Reply:
yep good shout, I'll remove all the unused reasons.